src/App.css

```diff
@@ -0,0 +1,159 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  color: white;
+}
+
+@font-face {
+  font-family: 'Lato';
+  src: url('./assets/Lato-Regular.ttf');
+}
+
+@font-face {
+  font-family: 'Lato';
+  src: url('./assets/Lato-Light.ttf');
+}
+
+@font-face {
+  font-family: 'Lato';
+  src: url('./assets/Lato-Bold.ttf');
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  background-color: rgb(255, 1, 213);
+  min-width: 800px;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+.nav-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 15vh;
+  padding: 1rem 15%;
+  background-color: rgb(253, 4, 158);
```

Comment:
- [ ] As required by the last CR please 🙏🏽 **select one main colour for the app** for a better user interface and match the original design 👇🏽 

> Students should select one main colour for the app and be consistent (it does not need to be the same colour as in the design guidelines).